lib/auth_middleware/setup.py

```diff
@@ -18,7 +18,7 @@
     name='auth-middleware',
     version='0.1.0',
     author='Frank Henard',
-    author_email='fhena2@illinois.edu',
+    author_email='fhena2@illinois.edu, ywkim@illinois.edu',
```

Comment:
```suggestion
    maintainer='Yong Wook Kim',
    maintainer_email='ywkim@illinois.edu',
```

Reply:
changed

Reply:
I meant to add you add you as maintainer and remove author_email. Please see my suggestion above.

```
maintainer='Yong Wook Kim',
maintainer_email='ywkim@illinois.edu',
```

Reply:
updated.